recipes/Open3D/meta.yaml

```diff
@@ -0,0 +1,125 @@
+{% set name = "Open3D" %}
+{% set version = "0.18.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/isl-org/Open3D/archive/v{{ version }}.tar.gz
+  sha256: 524ddeb7dc8aaed6dc5b272415df0c8ffcf7eff0816a84cfbdee56cdd6d3587a
+  patches:
+    - fix-tbb.patch
+    - fix-unzip.patch
+    - fix-curl.patch
+    - fix-error.patch
+    - fix-find-imgui.patch
+
+build:
+  number: 0
+  skip: True  # [win]
+  ignore_run_exports_from:
+    - libboost-devel
+    - libcblas
+    - minizip
+    - msgpack-c
+    - msgpack-cxx
+  run_exports:
+    - {{ pin_subpackage('open3d', max_pin='x.x.x') }}
```

Comment:
Do you want help splitting this package into dev and non-dev packages or you are OK will all users getting all artifacts and dependencies?

Reply:
I am ok with all users getting all artefacts/dependencies.